tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+    "compilerOptions": {
+        "lib": [
+            "es2015.iterable"
```

Comment:
I get a compilation error when I import the library, if I don't have this compiler option in my own tsconfig.json.

I suppose the best way would be to follow the StackOverflow suggestion: not implement that interface.